working/3616 - enum value shorthand/proposal-simple-lrhn.md

```diff
@@ -245,8 +250,8 @@ second operand.
 Examples of allowed comparisons:
 
 ```dart
-if (Endian.host == .big) ok!;
-if (Endian.host case == .big) ok!
+if (Endian.host == .big);       // ok!
+if (Endian.host case == .big);  // ok!
```

Comment:
This was deliberate. The `ok!` is a valid statement (or would be if it had the trailing `;` in the second line). It looks weird to have an `if` wih not body. Valid, but weird.
If the `ok!` doesn't look enough like a statement, then maybe change it to `isOK();`.

Reply:
Oh, haha, I didn't think about it being a null-assert of a variable named `ok`. :)